lib/Compiler.cpp

```diff
@@ -407,15 +407,26 @@ int SetCompilerInstanceOptions(CompilerInstance &instance,
   }
 
   clang::LangStandard::Kind standard;
```

Comment:
Could this enum be used directly in the option? On the one hand having a separate enum might be interesting for defining a clspv-specific flavor of opencl, but all those extra checks are in the frontend plugin and clang wouldn't know about it. So it might be better to reuse clang's enum if possible and change the default case to be an error for an unsupported kind.

Reply:
Yes, but that would mean effectively pulling Clang as a dependency to the passes library which I didn't feel comfortable with.